[2024-05-25T07:20:11.215+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-25T07:20:11.231+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sample.bash_run manual__2024-05-25T07:18:09.132003+00:00 [queued]>
[2024-05-25T07:20:11.234+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sample.bash_run manual__2024-05-25T07:18:09.132003+00:00 [queued]>
[2024-05-25T07:20:11.235+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 4
[2024-05-25T07:20:11.239+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): bash_run> on 2024-05-25 07:18:09.132003+00:00
[2024-05-25T07:20:11.244+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=812) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-25T07:20:11.245+0000] {standard_task_runner.py:63} INFO - Started process 813 to run task
[2024-05-25T07:20:11.244+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'sample', 'bash_run', 'manual__2024-05-25T07:18:09.132003+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/AirflowDeepLearningOperator.py', '--cfg-path', '/tmp/tmpccocp20s']
[2024-05-25T07:20:11.245+0000] {standard_task_runner.py:91} INFO - Job 6: Subtask bash_run
[2024-05-25T07:20:11.268+0000] {task_command.py:426} INFO - Running <TaskInstance: sample.bash_run manual__2024-05-25T07:18:09.132003+00:00 [running]> on host 803f731ab2bf
[2024-05-25T07:20:11.307+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='synarcs' AIRFLOW_CTX_DAG_ID='sample' AIRFLOW_CTX_TASK_ID='bash_run' AIRFLOW_CTX_EXECUTION_DATE='2024-05-25T07:18:09.132003+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-25T07:18:09.132003+00:00'
[2024-05-25T07:20:11.308+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-25T07:20:11.314+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-25T07:20:11.315+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'uname -a']
[2024-05-25T07:20:11.320+0000] {subprocess.py:86} INFO - Output:
[2024-05-25T07:20:11.322+0000] {subprocess.py:93} INFO - Linux 803f731ab2bf 5.15.49-linuxkit-pr #1 SMP PREEMPT Thu May 25 07:27:39 UTC 2023 aarch64 GNU/Linux
[2024-05-25T07:20:11.322+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-25T07:20:11.323+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-25T07:20:11.336+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=sample, task_id=bash_run, run_id=manual__2024-05-25T07:18:09.132003+00:00, execution_date=20240525T071809, start_date=20240525T072011, end_date=20240525T072011
[2024-05-25T07:20:11.382+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-25T07:20:11.392+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-25T07:20:11.393+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
